nilearn/decoding/decoder.py

```diff
@@ -876,3 +921,301 @@ def __init__(self, estimator='svr', mask=None, cv=10, param_grid=None,
             mask_strategy=mask_strategy, memory=memory,
             is_classification=False, memory_level=memory_level,
             verbose=verbose, n_jobs=n_jobs)
+
+
+class fREMRegressor(_BaseDecoder):
+    """ State of the art decoding scheme applied to usual regression estimators.
+
+    fREM uses an implicit spatial regularization through fast clustering and
+    aggregates a high number of estimators trained on various splits of the
+    training set, thus returning a very robust decoder at a lower computational
+    cost than other spatially regularized methods.[1]_.
+
+    Parameters
+    -----------
+    estimator: str, optional
+        The estimator to choose among: 'ridge', 'ridge_regressor', and 'svr'.
+        Note that the 'ridge' and 'ridge_regressor' correspond to the same
+        estimator. Default 'svr'.
+
+    mask: filename, Nifti1Image, NiftiMasker, or MultiNiftiMasker, optional
+        Mask to be used on data. If an instance of masker is passed,
+        then its mask and parameters will be used. If no mask is given, mask
+        will be computed automatically from provided images by an inbuilt
+        masker with default parameters. Refer to NiftiMasker or
+        MultiNiftiMasker to check for default parameters. Default None
+
+    cv : int or cross-validation generator, optional (default 30)
+        If int, number of shuffled splits returned, which is usually
+        the right way to train many different classifiers. A good trade-off between
+        stability of the aggregated model and computation time is 50 splits.
+        Can also be a cross-validation generator.
+
+    param_grid: dict of str to sequence, or sequence of such. Default None
+        The parameter grid to explore, as a dictionary mapping estimator
+        parameters to sequences of allowed values.
+
+        None or an empty dict signifies default parameters.
+
+        A sequence of dicts signifies a sequence of grids to search, and is
+        useful to avoid exploring parameter combinations that make no sense
+        or have no effect. See scikit-learn documentation for more information,
+        for example: https://scikit-learn.org/stable/modules/grid_search.html
+
+    clustering_percentile: int, float, optional, in the closed interval [0, 100]
+        Used to perform a fast ReNA clustering on input data as a first step
+        of fit. It agglomerates similar features together to reduce their number
+        by this percentile. ReNA is typically efficient at reducing the number
+        of features by 10. Default: 10.
```

Comment:
I find this formulation a bit  ambiguous. Let me suggest:
"""
ReNA is typically efficient for cluster_percentile equal to 10. Default: 10.
"""